hypershift-operator/controllers/hostedcluster/hostedcluster_controller_test.go

```diff
@@ -3062,3 +3062,54 @@ func (k *kubevirtInfraClientMapMock) GetClient(_ string) *kvinfra.KubevirtInfraC
 func (*kubevirtInfraClientMapMock) Delete(_ string) {
 	// interface's empty implementation
 }
+
+func Test_isIPv4(t *testing.T) {
```

Comment:
Every test in this file follows TestIsIPV4 convention, any reason to deviate from that?

Reply:
It's more like "isIPv4" implies private function meanwhile "IsIPv4" more a public one. But I think I will move it  the test and the function to the support module because It will be used by CPO and HO at some point (this will be used in a new issue I'm working on now). So I think that make sense to move it to a common place.